torch/csrc/jit/frontend/ir_emitter.cpp

```diff
@@ -3393,30 +3393,32 @@ struct to_ir {
         ->output();
   }
 
-  Value* emitSubscript(const Subscript& subscript) {
+  std::shared_ptr<SugaredValue> emitSubscript(const Subscript& subscript) {
```

Comment:
nit: `SugaredValuePtr` ?  used twice as much in this file as `std::shared_ptr<SugaredValue>`...

Reply:
I'm going to leave as is rather than hit another 6 hour diff test, but maybe we can change them all to `SugaredValuePtr` in a follow up